core-java/src/main/java/org/spine3/server/BoundedContext.java

```diff
@@ -165,7 +166,7 @@ private void shutDownRepositories() {
      * <p>For instances of {@link AggregateRepository} an instance of {@link AggregateStorage} is created
      * and assigned.
      *
-     * <p>For other types of repositories an instance of {@link org.spine3.server.storage.EntityStorage} is
+     * <p>For other types of repositories an instance of {@link EntityStorage} is
```

Comment:
Let's discuss the way we work with storages. I suppose, we can simplify this and make it more transparent for the developer. Now assigning a storage is a hidden thing, which requires having a storage factory.
Instead, we can accept fully built instance of a repository. This way we would be able adding new kinds of repositories _without_ the need to modify StorageFactory API.